### PR TITLE
fix: "turn on automated checks" text is outdated

### PR DESCRIPTION
**Pull request by smoralesd**
#### Description of changes

Using the cards ui feature flag to make the text different when the automated checks are disabled.

**Fix for cards ui**
01 - disabled message cards ui

**Not changed for details list**
02 - disabled message details list

#### Pull request checklist

- [x] Addresses an existing issue: #1754
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [x] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/DetailsView/components/issues-table.tsx [hunk 1]
```diff
@@ -183,11 +183,21 @@ export class IssuesTable extends React.Component<IssuesTableProps> {
         const isCardsUIEnabled = this.props.featureFlags[FeatureFlags.universalCardsUI];
         const commandBar = !isCardsUIEnabled ? this.renderCommandBar() : null;
 
+        const disabledMessage = isCardsUIEnabled ? (
+            <span>
+                Use the <Markup.Term>start over</Markup.Term> button to scan the target page.
```
Comment (dbjorge) [inline src/DetailsView/components/issues-table.tsx]: Should this be "Start over" for consistency with the button text?

File: src/tests/unit/tests/DetailsView/components/__snapshots__/issues-table.test.tsx.snap [hunk 2]
```diff
@@ -27,11 +27,13 @@ exports[`IssuesTableTest automated checks disabled 1`] = `
         className="details-disabled-message"
         role="alert"
       >
-        Turn on 
-        <Term>
-          Automated checks
-        </Term>
-         to see a list of failures.
+        <span>
```
Comment (dbjorge) [inline src/tests/unit/tests/DetailsView/components/__snapshots__/issues-table.test.tsx.snap]: Should there be a snapshot test that verifies the new text?

File: src/DetailsView/components/issues-table.tsx (hunk 1 above)
Comment (dbjorge) [reply src/DetailsView/components/issues-table.tsx]: (with capital S)

Comment (smoralesd) [reply src/DetailsView/components/issues-table.tsx]: updated

File: src/tests/unit/tests/DetailsView/components/__snapshots__/issues-table.test.tsx.snap (hunk 2 above)
Comment (smoralesd) [reply src/tests/unit/tests/DetailsView/components/__snapshots__/issues-table.test.tsx.snap]: yeap... I also notices when change the case of 's'

**Pull request merged**